action-types.yml

```diff
@@ -0,0 +1,24 @@
+# This file provides types for "GitHub Actions Kotlin DSL".
+# See https://github.com/krzema12/github-actions-kotlin-dsl
+
+inputs:
+  repo-token:
+    type: string
+  dry-run:
+    type: boolean
+  ignore:
+    type: string
```

Comment:
I think it makes more sense to define this as a [list](https://github.com/typesafegithub/github-actions-typing/tree/ef699fb69fb11ba9b1520f57b257c90ab9739dfc#list) type. Per the [example configuration](https://github.com/ericcornelissen/svgo-action/tree/6945044d2d7dcfe165792e28094d2ba7443d0513#configure-the-action), this is expected to be a list of globs separated by newlines.

Reply:
Resolved with 32344aab6fdddbbcdeb477070160ea319997d2a9